src/components/ISPFilter/ISPFilter.css

```diff
@@ -0,0 +1,48 @@
+.isp-filter {
+  background-color: var(--white);
+  padding-bottom: 1.875rem;
+  font-family: var(--f-family-inter);
+}
+
+.isp-label {
+  font-size: 1rem;
+  color: var(--grey-200);
+}
+
+.chip-buttons {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1.4375rem;
+  margin-bottom: 1.25rem;
+  padding-left: 1.1875rem;
+}
+
+.chip {
+  width: 4.8rem;
```

Comment:
remove this width. let the chip expand according to the text. so just just make sure the padding needed for the button is correct.